LEAF_Request_Portal/form.php

```diff
@@ -495,6 +495,8 @@ public function getIndicator($indicatorID, $series, $recordID = null, $parseTemp
         $form[$idx]['isWritable'] = $this->hasWriteAccess($recordID, $data[0]['categoryID']);
         $form[$idx]['isMasked'] = isset($data[0]['groupID']) ? $this->isMasked($data[0]['indicatorID'], $recordID) : 0;
         $form[$idx]['sort'] = $data[0]['sort'];
+        $form[$idx]['has_code'] = ($data[0]['html'] != null && trim($data[0]['html']) != '') ||
```

Comment:
Checking if `$data[0]['groupID'])` is not null is redundant here; `trim($var)` returns the empty string when `$var` is null.

Reply:
I removed the null checks